src/Umbraco.Web.UI.Client/src/packages/documents/documents/item/document-item-data-resolver.ts

```diff
@@ -2,12 +2,26 @@ import { UmbDocumentVariantState } from '../types.js';
 import type { UmbDocumentItemModel } from './types.js';
 import { UmbControllerBase } from '@umbraco-cms/backoffice/class-api';
 import type { UmbControllerHost } from '@umbraco-cms/backoffice/controller-api';
+import type { UmbEntityFlag } from '@umbraco-cms/backoffice/entity';
 import type { DocumentVariantStateModel } from '@umbraco-cms/backoffice/external/backend-api';
-import { UmbBooleanState, UmbObjectState, UmbStringState } from '@umbraco-cms/backoffice/observable-api';
+import {
+	type Observable,
+	UmbArrayState,
+	UmbBooleanState,
+	UmbObjectState,
+	UmbStringState,
+} from '@umbraco-cms/backoffice/observable-api';
 import { type UmbVariantContext, UMB_VARIANT_CONTEXT } from '@umbraco-cms/backoffice/variant';
 
 type UmbDocumentItemDataResolverModel = Omit<UmbDocumentItemModel, 'parent' | 'hasChildren'>;
 
+function IsVariantsInvariant(variants: Array<{ culture: string | null }>): boolean {
```

Comment:
Can we camelCase these function names so only class names are PascalCase?

Reply:
This is my Flash Days legacy. we always had static things with PascalCase.. but I think your right that is properly how we do it in general.